client/src/components/PostCreationPage.tsx

```diff
@@ -9,53 +9,66 @@ import Stack from '@mui/material/Stack';
 import Divider from '@mui/material/Divider';
 import Input from '@mui/material/Input';
 import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper'
+import Grid from '@mui/material/Grid';
+import TabContext from '@mui/lab/TabContext';
+import TabList from '@mui/lab/TabList';
+import Tab from '@mui/material/Tab';
+import TabPanel from '@mui/lab/TabPanel';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import { Tags } from '@prisma/client';
 
 const PostCreationPage = (): ReactElement => {
   const navigate = useNavigate();
-  const [title, setTitle]: [string, Function] = useState('');
+  const [title, setTitle]: [string, Function] = useState('# ');
   const [body, setBody]: [string, Function] = useState('');
   const [titleFieldTooltip, setTitleFieldTooltip] = useState(false);
   const [bodyFieldTooltip, setBodyFieldTooltip] = useState(false);
-  const [img, setImg]: [any, Function] = useState();
+  // const [img, setImg]: [any, Function] = useState();
   const [cantSubmit, setCantSubmit]: [boolean, Function] = useState(false);
   const [repo, setRepo]: [{link: string, files: { path: string; contents: string }[]},Function] = useState({link:'', files:[]});
+  const [currentTab, setCurrentTab] = useState('0');
+  const [allPostTags, setAllPostTags]: [Tags[]|null, Function] = useState(null);
+  const allPostTagsREF = useRef(allPostTags);
+  const [selectedTags, setSelectedTags]: [Tags[], Function] = useState([]);
 
+  useEffect(() => {
+    axios.get('/api/tags/all/post')
+      .then(({data}) => {
+        setAllPostTags(data);
+      })
+  }, [allPostTagsREF])
+
+  const handleTabChange = (e: React.SyntheticEvent, newValue: string) => {
+    setCurrentTab(newValue);
+  }
 
   const handleTextInput = (
     e:
       | React.ChangeEvent<HTMLInputElement>
       | React.ChangeEvent<HTMLTextAreaElement>,
   ): void => {
-    let setStateFunc: Function;
-    if (e.target.name === 'title') {
-      if (titleFieldTooltip) {
-        setTitleFieldTooltip(false);
-      }
-      setStateFunc = setTitle;
-    } else if (e.target.name === 'body') {
-      if (bodyFieldTooltip) {
-        setBodyFieldTooltip(false);
-      }
-      setStateFunc = setBody;
-    } else {
-      return;
+    switch (e.target.name) {
+      case 'title': {setTitle(`# ${e.target.value}`); break;}
+      case 'body': {setBody(e.target.value); break;}
     }
-    setStateFunc(e.target.value);
   };
 
-  const handleFile = (e: React.ChangeEvent<HTMLInputElement>): void => {
-    console.log(e.target.files);
-    setImg(e.target.files![0]);
-  };
+  // const handleFile = (e: React.ChangeEvent<HTMLInputElement>): void => {
+  //   setImg(e.target.files![0]);
+  // };
```

Comment:
Should be removed if not used

Reply:
This is only not in use because the image display is not ready. When i fix that I dont want to go back and re-implement all the logic for it, so it is commented out for the time being